### ISSUE TITLE
Allow repositories to be marked as non-star (:thumbsdown:) 👎

### ISSUE THREAD

**Issue by jhinrichsen** [dear-github/dear-github#359]
As a user, i can `Star` repositories that i like. `Unstar` will remove this `Star`, but i don't have the possibility to express my dislike for a repository.

Not sure if this is the right place to leave this request for improvement.

**Comment by Kamaropoulos** [comment]
I honestly don't get what would that achieve, other than discouraging the author(s) and slowing down development on something that could possibly turn out to be interesting in the future.
Personally I believe in constructive criticism, so instead of saying to someone how bad work they are doing, why not open an issue and discuss why you think the repo is not good in the first place? This way, instead of just hurting the other person's dealings, you could help them make it better and more useful to everyone!

-1, change my mind 😛

**Comment by jhinrichsen** [comment]
I totally agree, constructive criticism is the way to go. On the other hand, most websites that accept feedback (Facebook, Youtube) also allow you to express your dislike. In the context of open source development, we are increasingly faced with the problem of external dependencies, e.g. Go modules or Rust crates.  I sometimes invest a serious amount of work to come up with "the best as of now" library that solves a given problem, e.g. a HTTP client libraries. There's a ton of HTTP client libraries out there, ranging from some student's first try to bullet proof industrial strength implementations that support proxies, NTLM, you name it. Sometimes it is easy to separate the good from the bad, sometimes its hard and takes a serious amount of time _for each and every user over and over again_. A dislike and a comment could stop wasting this research time.

Your point w/ regard to raising an issue is absolutely valid. But sometimes repositories are completely bent out of shape, and there's no way of requesting a 'hey, get this right', because many repositories are already abandonded. The author has moved on, sometimes leaving a note, sometimes not. If a repository that has been abandonded long time ago, uses obsolete syntax, does not compile, and does not say so, why not leave a mark for others?

And my last point - me as an author/ owner of a repository, i very welcome a dislike as a feedback instead of no feedback. This way i can reach out and contact the disliker, understand why other people will or won't use my stuff. Most users in research for a solution will just use another provider instead of opening an issue. Negative feedback is better than no feedback.
